tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -132,8 +132,8 @@ def get(self, request, *args, **kwargs):
 
                 return response_redirect(user, id_token)
 
-        except Exception as e:  # pragma: no cover
-            logger.info(f"Error attempting to login/registeruser:  {e} at...")
+        except Exception as e:
+            logger.exception(f"Error attempting to login/registeruser:  {e} at...")
```

Comment:
Please change `Error attempting to login/registeruser` to `Error attempting to login/register user`